src/Elasticsearch.Net.VirtualizedCluster/Audit/Auditor.cs

```diff
@@ -195,13 +198,13 @@ private Auditor AssertAuditTrails(ClientCall callTrace, int nthCall)
 		public void VisualizeCalls(int numberOfCalls)
 		{
 			var cluster = _cluster ?? Cluster();
-			var messages = new List<string>(numberOfCalls * 2);
-			for (var i = 0; i < numberOfCalls; i++)
+			var messages = new List<string>((numberOfCalls -1) * 2);
+			for (var i = 0; i <= numberOfCalls - 1; i++)
```

Comment:
I'm missing the context of how this is used - can you explain what the logical change is in using `numberOfCalls - 1` (and also doing `<=` in addition)?

Reply:
None, this should be reverted 👍.